hooks/hookstage/processedauctionrequest_mutations.go

```diff
@@ -32,25 +32,62 @@ func (c ChangeSetProcessedAuctionRequest[T]) castPayload(p T) (*openrtb_ext.Requ
 	return nil, errors.New("failed to cast ProcessedAuctionRequestPayload")
 }
 
-func (c ChangeBidders[T]) Update(impIdToBidders map[string]map[string]json.RawMessage) {
```

Comment:
We still have `MutationUpdate` in `hooks/hookstage/mutation.go` and references to `MutationUpdate` in `hooks/hookstage/rawbidderresponse_mutations.go` and `modules/fiftyonedegrees/devicedetection/hook_raw_auction_request.go`:
```
  5 const (
  6     MutationAdd MutationType = iota
  7     MutationUpdate
  8     MutationDelete
  9 )
hooks/hookstage/mutation.go
```
Should we keep the `changeBidders.Update(impIdToBidders map[string]map[string]json.RawMessage)` function , even if it does nothing, here in `hooks/hookstage/processedauctionrequest_mutations.go` just to be on par with other hooks?

Reply:
No, we don't need to have empty "Update". This constant is not an interface to implement, it's a list of types to return from the "AddMutation".